joe-spring-boot/src/main/java/com/example/joespringboot/GreetingController.java

```diff
@@ -13,8 +13,8 @@
     private final AtomicLong counter = new AtomicLong();
 
     @GetMapping("/greeting")
-    public Greeting greeting(@RequestParam(value = "name", defaultValue = "World") String name){
-        return new Greeting(counter.incrementAndGet(),String.format(template, name));
+    public Greeting greeting(@RequestParam(value = "name", defaultValue = "World") String name) {
+        return new Greeting(counter.incrementAndGet(), String.format(template, name));
```

Comment:
10/10 excellent whitespace

Reply:
nyost nyexcellent. thank you! yes, i often use the formatting shortcut in intellij (though i've not amended any of the style settings. hopefully the defauts will suffice